DTXManiaプロジェクト/コード/スコア、曲/CDTX.cs

```diff
@@ -7699,61 +7520,6 @@ private bool t入力_行解析_WAVPAN_PAN( string strコマンド, string strパ
 
 			return true;
 		}
-		private bool t入力_行解析_WAVVOL_VOLUME( string strコマンド, string strパラメータ, string strコメント )
```

Comment:
Here's the actual deletion of WAVVOL and VOLUME handling, as opposed to the call site noted earlier.